.github/workflows/ci.yml

```
@@ -42,14 +81,78 @@ jobs:
           fi
 
       - name: Lint (clang-tidy)
+        shell: bash
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:12: Double quote to prevent globbing and word splitting

(shellcheck)